lib/Utils/Architectures.pm

```diff
@@ -37,6 +37,8 @@ use constant {
           is_aarch64
           is_arm
           is_ppc64le
+          is_orthos_machine
+          is_covered_machine
```

Comment:
better indicate who or what covered machine from function name, or it will be too general

Reply:
is_covered_machine is just to check whether it is covered, "covered" here means everything. You can go to function description to know the details. So I opt to no.

Reply:
`is_covered_machine` is in my understanding not any better than `is_valid_machine`. Just think of any outside newcome reading `lib/Utils/Architectures.pm`. Methods like `is_orthos_machine` and `is_aarch64` explain in the name already to what this applies but `is_covered_machine` very much depends on what "covered" means in this context. Same as in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/10946#discussion_r484669608 this is still too generic

Reply:
I think it is enough. If you think there is better one, please suggest your preferred subroutine name.

Reply:
How about `is_supported_suse_domain`?

Reply:
OK. Done.